Mathlib/MeasureTheory/Measure/Haar/Basic.lean

```diff
@@ -698,6 +698,12 @@ theorem haarMeasure_unique (μ : Measure G) [SigmaFinite μ] [IsMulLeftInvariant
 #align measure_theory.measure.haar_measure_unique MeasureTheory.Measure.haarMeasure_unique
 #align measure_theory.measure.add_haar_measure_unique MeasureTheory.Measure.addHaarMeasure_unique
 
+@[to_additive]
+theorem  haarMeasure_eq (K₀ K₁ : TopologicalSpace.PositiveCompacts G) :
```

Comment:
I think this lemma could do with a docstring; the statement looks quite strange to me!